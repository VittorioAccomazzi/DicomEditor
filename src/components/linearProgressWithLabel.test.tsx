
import React from 'react';
import { cleanup, render } from '@testing-library/react'
import LinearProgressBarWithLabel from './linearProgressWithLabel'

beforeEach(() => {
  // setup a DOM element as a render target
});

afterEach(()=>{
  cleanup()
  jest.clearAllMocks()
})

test('rendering',()=>{
    const {container} =  render(<LinearProgressBarWithLabel value={50}/>)
    expect(container.innerHTML).toMatchSnapshot()
})