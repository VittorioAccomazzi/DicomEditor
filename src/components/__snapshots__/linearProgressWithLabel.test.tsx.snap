// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `"<div class=\\"MuiBox-root MuiBox-root-1\\"><div class=\\"MuiBox-root MuiBox-root-2\\"><div class=\\"MuiLinearProgress-root WithStyles(ForwardRef(LinearProgress))-root-3 MuiLinearProgress-colorPrimary MuiLinearProgress-determinate\\" role=\\"progressbar\\" aria-valuenow=\\"50\\" aria-valuemin=\\"0\\" aria-valuemax=\\"100\\"><div class=\\"MuiLinearProgress-bar WithStyles(ForwardRef(LinearProgress))-bar-4 MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Determinate\\" style=\\"transform: translateX(-50%);\\"></div></div></div><div class=\\"MuiBox-root MuiBox-root-5\\"><p class=\\"MuiTypography-root text MuiTypography-body1\\">50%</p></div></div>"`;
